src/utils.rs

```diff
@@ -57,3 +57,18 @@ pub fn effective_value(output: &OutputGroup, feerate: f32) -> u64 {
         .value
         .saturating_sub(calculate_fee(output.weight, feerate))
 }
+
+// Returns the weight of the output group(s)
+pub fn calculate_base_weight_btc(output_weight: u64) -> u64 {
+    // VERSION_SIZE: 4 bytes - 16 WU
+    // SEGWIT_MARKER_SIZE: 2 bytes - 2 WU
+    // NUM_INPUTS_SIZE: 1 byte - 4 WU
+    // NUM_OUTPUTS_SIZE: 1 byte - 4 WU
+    // NUM_WITNESS_SIZE: 1 byte - 1 WU
+    // LOCK_TIME_SIZE: 4 bytes - 16 WU
+    // OUTPUT_VALUE_SIZE: variable
+
+    // Total default: (16 + 2 + 4 + 4 + 1 + 16 = 43 WU + variable) WU
```

Comment:
Why the return value does not have these weights?

Reply:
https://docs.rs/bitcoin/latest/src/bitcoin/blockdata/transaction.rs.html#599-602